src/bootman/kernel.c

```diff
@@ -607,6 +683,13 @@ bool boot_manager_remove_kernel_internal(const BootManager *manager, const Kerne
                 }
         }
 
+        /* Remove the source initrd in non-fatal fashion */
+        if (kernel->initrd_file && unlink(kernel->initrd_file) < 0) {
```

Comment:
Do you actually want to have this auto clean up the user configured initrds in /etc?